File: source/backend/projects/Pulse.Api/Models/Sessions/SessionCheckinDetailsDto.cs
namespace Pulse.Api.Models.Sessions
{
    using System;
    using System.ComponentModel.DataAnnotations;
    using Data;

    public class SessionCheckinDetailsDto
    {
        [Required]
        public Guid SessionCheckinId { get; set; }

        [Required]
        public Guid SessionId { get; set; }

        [Required]
        public CheckinType CheckinType { get; set; }

        public SessionQuestionDetailsDto[] Questions { get; set; }
    }
}

File: source/frontend/projects/pulse/src/app/api/models/session-checkin-details-dto.ts
/* tslint:disable */
/* eslint-disable */
import { CheckinType } from './checkin-type';
import { SessionQuestionDetailsDto } from './session-question-details-dto';
export interface SessionCheckinDetailsDto {
  checkinType: CheckinType;
  questions?: null | Array<SessionQuestionDetailsDto>;
  sessionCheckinId: string;
  sessionId: string;
}
